src/Angor/Client/Pages/Invest.razor

```diff
@@ -531,12 +539,19 @@ else
     {
         if (Investment.IsSeeder)
         {
-            var minSeederAmount = 2;
+            var minSeederAmount = 0.01m;
```

Comment:
Seeder is not implemented yet (the new name is lead investor) so don't do anything here it will just be confusing)